app/src/main/java/gov/hhs/cdc/trustedintermediary/external/javalin/App.java

```diff
@@ -13,6 +14,8 @@ public static void main(String[] args) {
 
         app.get("/health", ctx -> ctx.result("Operational"));
 
+        ApplicationContext.register(TiLogger.class, TiLogger.getLogger());
```

Comment:
We should register the logger under the interface name. So it would become `MyLogger.class`. Or just `Logger.class` if you take my rename suggestion I made in another comment.

Reply:
I love the idea.